pkg/config/config.go

```diff
@@ -1,16 +1,22 @@
 package config
 
-import "os"
+import (
+	"os"
+	"sync"
+)
 
 var urldbConn string
+var m = &sync.Mutex{}
 
 func SetURLDBConn(urlconnection string) {
 	urldbConn = urlconnection
```

Comment:
This should probably use the mutex as well ?

Reply:
Well It's used only once throughout whole program and does not cause race condition, do you think that nevertheless mutex should be used for setting value here as well?

Reply:
@alaingilbert